### PR TITLE
MRG, ENH: Use sosfiltfilt in raw plotting

### PR DESCRIPTION
**Pull request by larsoner**
Use more stable high-order filtering in `raw.plot`, and add `--verbose` option to `mne browse_raw` so that this command works (and reports filter info):
```
mne browse_raw clouds_2ae/raw_fif/clouds_2ae_01_raw.fif --allowmaxshield --lowpass 30 --highpass 1 --filtorder 8 --verbose
...
Setting up band-pass filter from 1 - 30 Hz

IIR filter parameters
---------------------
Butterworth bandpass zero-phase (two-pass forward and reverse) non-causal filter:
- Filter order 32 (effective, after forward-backward)
- Cutoffs at 1.00, 30.00 Hz: -6.02, -6.02 dB
```
(yes I know this filter is a bit crazy, but it still should not break things)

### CODE REVIEW

**Comment by larsoner** [comment]
@drammock if you are still in a reviewing merge this one's good to go, too

File: mne/viz/tests/test_raw.py [hunk 1]
```diff
@@ -337,22 +337,21 @@ def test_plot_annotations():
...
-        raw.plot(lowpass=raw.info['sfreq'] / 2.)
+    with pytest.raises(ValueError, match='lowpass.*Nyquist'):
+        raw.plot(lowpass=raw.info['sfreq'] / 2., filtorder=filtorder)
     with pytest.raises(ValueError, match='highpass must be > 0'):
-        raw.plot(highpass=0)
-    with pytest.raises(ValueError, match=r'lowpass \(1\) must be > highpass'):
-        raw.plot(lowpass=1, highpass=1)
-    with pytest.raises(ValueError, match=r'filtorder \(-1\) must be >= 0'):
+        raw.plot(highpass=0, filtorder=filtorder)
+    with pytest.raises(ValueError, match=r'Filter order must be'):
```
Comment (drammock) [inline mne/viz/tests/test_raw.py]: not sure why you need an `r` string here.

Comment (larsoner) [reply mne/viz/tests/test_raw.py]: Errant copy-paste cruft

**Comment by drammock** [comment]
thx @larsoner

**Pull request merged**